frontend/src/component/admin/network/Network.tsx

```diff
@@ -28,17 +31,31 @@ const tabs = [
         label: 'Connected Edges',
         path: '/admin/network/connected-edges',
     },
+];
+
+const seatModelTabs = [
     {
         label: 'Data Usage',
         path: '/admin/network/data-usage',
     },
 ];
 
+const consumptionModelTabs = [
+    {
+        label: 'Backend Connections',
+        path: '/admin/network/backend-connections',
+    },
+];
+
 export const Network = () => {
     const { pathname } = useLocation();
     const edgeObservabilityEnabled = useUiFlag('edgeObservability');
+    const consumptionModelEnabled = useUiFlag('consumptionModel');
+    const allTabs = consumptionModelEnabled
```

Comment:
choose which tabs to show in seat mode and consumption mode. Consumption mode will get one more tab in another PR